app/src/main/java/co/condorlabs/customcomponents/simplecamerax/CameraActivity.kt

```diff
@@ -122,7 +151,8 @@ class CameraActivity : AppCompatActivity(), SimpleCameraXFragment.OnCameraXListe
         imageCaptureError: ImageCapture.ImageCaptureError,
         message: String,
         cause: Throwable?
-    ) {}
+    ) {
```

Comment:
ca you evaluate if this method can be implemented or deleted, for not violate the segregation interface principle